Targets/STM32F4/STM32F4.h

```diff
@@ -333,4 +333,25 @@ TinyCLR_Result STM32F4_Display_DrawBuffer(const TinyCLR_Display_Provider* self,
 TinyCLR_Result STM32F4_Display_WriteString(const TinyCLR_Display_Provider* self, const char* buffer);
 int32_t STM32F4_Display_GetWidth(const TinyCLR_Display_Provider* self);
 int32_t STM32F4_Display_GetHeight(const TinyCLR_Display_Provider* self);
-TinyCLR_Display_InterfaceType STM32F4_Display_GetType(const TinyCLR_Display_Provider* self);
\ No newline at end of file
+TinyCLR_Display_InterfaceType STM32F4_Display_GetType(const TinyCLR_Display_Provider* self);
+
+const TinyCLR_Api_Info* STM32F4_Can_GetApi();
```

Comment:
Match the order in STM32F4.h, this should be between ADC and DAC (alphabetical essentially)